pkg/repo/repo.go

```diff
@@ -46,3 +46,21 @@ func (repo *Repo) SaveMessages(id string, m model.Messages) error {
 
 	return nil
 }
+
+func (repo *Repo) LoadMessages(id string) error {
+	err := repo.db.View(func(txn *badger.Txn) error {
+		item, err := txn.Get([]byte(id))
+		if err != nil {
+			return fmt.Errorf("getting key/value pair: %w", err)
+		}
+		err = item.Value(func(val []byte) error {
+			return fmt.Errorf("getting the value of the item: %w", err)
```

Comment:
error should be returned when not found, in this case error is always returned, and also You dont use `val` parameter